src/values/basic_value_use.rs

```diff
@@ -18,7 +18,7 @@ impl<'ctx> BasicValueUse<'ctx> {
         BasicValueUse(use_, PhantomData)
     }
 
-    /// Gets the next use of an `InstructionValue` or `BasicValue` if any.
+    /// Gets the next use of a `BasicValue` if any.
```

Comment:
Isn't the original comment still accurate? It's a use of either an instruction or basic value, no?

Reply:
Or maybe it should say `AnyValueEnum`?

Reply:
No, you're right, I shouldn't have changed this at all. The only way to create a BasicValueUse is with the method either on BasicValue or on InstructionValue and that hasn't changed.